modules/core/distribution/src/assembly/filter.properties

```diff
@@ -25,3 +25,5 @@ carbon.version=4.4.9
 default.server.role=IoTServer
 cep.server.role=ComplexEventProcessor
 das.server.role=DataAnalyticsServer
+emm.analytics.role=EMMAnalytics
```

Comment:
We are using IoTServer role in gadgets and dashboards, hence remove this with 090da7941920d7cd57a473b3abe894d4901eb095